examples/crewai/1. tool-example/agent_crewai.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import os
+from textwrap import dedent
+
+from arcadepy import Arcade
+from crewai import Agent, Crew, Task
+from crewai_arcade.manager import CrewAIToolManager
+from dotenv import load_dotenv
+from langchain_openai import ChatOpenAI
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+load_dotenv()
+
+arcade_client = Arcade()
+user_id = os.getenv("ARCADE_USER_ID")
+
+manager = CrewAIToolManager(arcade_client, user_id)
+# Retrieve the tools from the specified toolkit
+tools = manager.get_tools(toolkits=["google"])
```

Comment:
@lgesuellip  Google calendar tools require authorization. I suspect you have already authorized Arcade for the google calendar tool with the cloud engine, so you likely did not run into this issue. You can use the `arcade_client.tools.authorize` function to authorize a specific tool. See [this example](https://github.com/ArcadeAI/arcade-ai/blob/main/examples/call_a_tool_directly_with_auth.py) for authorizing a specific Google tool.

I also recommend running a [local engine](https://docs.arcade-ai.com/home/install/overview) when developing so that authorizations are reset on each engine startup.

Reply:
Hey Eric, thank you for the feedback.

The tool includes logic to verify authentication, similar to the langgraph integration. If authentication is required but hasn’t been completed, it returns a link for the user to authenticate (https://github.com/ArcadeAI/arcade-ai/blob/6f92a6d4b29a83f96aab579d7383c2b0108669e2/contrib/crewai/crewai_arcade/manager.py#L14)

What do you think?
Would you recommend adding any extra logic to the example, such as waiting until the authorization process is completed?